File: platforms/android/stato/src/main/java/com/facebook/stato/plugins/fresco/FrescoStatoDebugPrefHelper.java
// Copyright 2004-present Facebook. All Rights Reserved.

package com.facebook.stato.plugins.fresco;

public interface FrescoStatoDebugPrefHelper {

  interface Listener {
    void onEnabledStatusChanged(boolean enabled);
  }

  void setDebugOverlayEnabled(boolean enabled);

  boolean isDebugOverlayEnabled();

  void setDebugOverlayEnabledListener(Listener l);
}

File: platforms/android/android-stato/src/main/kotlin/org/stato/plugins/fresco/FrescoStatoDebugPrefHelper.kt
// Copyright 2004-present Facebook. All Rights Reserved.

package org.stato.plugins.fresco

interface FrescoStatoDebugPrefHelper {

  var isDebugOverlayEnabled: Boolean

  interface Listener {
    fun onEnabledStatusChanged(enabled: Boolean)
  }

  fun setDebugOverlayEnabledListener(l: Listener)
}
